Gemfile

```diff
@@ -19,6 +19,7 @@ gem "sidekiq-unique-jobs", git: "https://github.com/alphagov/sidekiq-unique-jobs
 
 group :development, :test do
   gem "byebug" # Comes standard with Rails
+  gem 'climate_control', '~> 0.2.0'
```

Comment:
It would be good to be consistent and use double quotation marks.

Reply:
Updated and fixed up.